kolibri/core/assets/src/vue/progress-icon/index.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+
+  <!--NOT STARTED-->
+  <span v-if="isNotStarted" class="wrapper">
+    <svg src="./progress-icons/notstarted.svg"></svg>
+    <span v-if="showtext" class="text">
+      <span v-if="isExercise">{{ $tr('not') }} {{ $tr('started') }}</span>
+      <span v-if="isVideo">{{ $tr('not') }} {{ $tr('watched') }}</span>
+      <span v-if="isAudio">{{ $tr('not') }} {{ $tr('listened') }}</span>
+      <span v-if="isDocument">{{ $tr('not') }} {{ $tr('viewed') }}</span>
+    </span>
+  </span>
+
+  <!--IN PROGRESS-->
+  <span v-if="isInProgress" class="wrapper">
+    <svg src="./progress-icons/inprogress.svg"></svg>
+    <span v-if="showtext" class="text">
+      <span>{{ $tr('inProgress') }}</span>
+    </span>
+  </span>
+
+  <!--COMPLETE-->
+  <span v-if="isComplete" class="wrapper">
+    <svg src="./progress-icons/complete.svg"></svg>
+    <span v-if="showtext" class="text">
+      <span v-if="isExercise">{{ $tr('mastered') }}</span>
+      <span v-if="isVideo">{{ $tr('Watched') }}</span>
+      <span v-if="isAudio">{{ $tr('listened') }}</span>
+      <span v-if="isDocument">{{ $tr('viewed') }}</span>
+    </span>
+  </span>
+
+  <!--TODO: ADD VIEWED AND NOT VIEWED ICONS?-->
+
+</template>
+
+
+<script>
+
+  const Constants = require('kolibri.coreVue.vuex.constants');
+  const values = require('lodash.values');
+
+  module.exports = {
+    $trNameSpace: 'progress-icon',
+    $trs: {
+      // TODO: Add these to Constants?
```

Comment:
generally we haven't been adding $tr identifiers to constants because they should be scoped to a single component file